### PR TITLE
updated shared lib to support configurable resource api versions

### PR DESCRIPTION
**Pull request by vgenev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,40 +54,41 @@
...
+    "@mojaloop/central-services-shared": "11.3.3",
     "@mojaloop/event-sdk": "10.6.0",
     "blipp": "4.0.1",
-    "commander": "6.0.0",
+    "commander": "6.1.0",
     "docdash": "1.2.0",
     "hapi-openapi": "2.0.2",
-    "hapi-swagger": "13.0.2",
+    "hapi-swagger": "14.0.0",
+    "joi": "^17.2.1",
```
Comment (rmothilal) [inline package.json:71]: Why has JOi been added?

**Review by rmothilal — CHANGES_REQUESTED** [review]
Please see comment

File: package.json (hunk 1 above)
Comment (vgenev) [reply package.json:71]: happy-swagger cannot find it

Comment (vgenev) [reply package.json:71]: and carrot will be removed

**Review by rmothilal — APPROVED** [review]
happy

**Pull request merged**